server/core/region.go

```diff
@@ -403,6 +403,14 @@ func (r *RegionInfo) GetReplicationStatus() *replication_modepb.RegionReplicatio
 	return r.replicationStatus
 }
 
+// GetReadQPS returns the region's qps by type.
+func (r *RegionInfo) GetReadQPS() uint64 {
+	if r.queryStats == nil {
+		return 0
+	}
+	return r.queryStats.Get + r.queryStats.Coprocessor + r.queryStats.Scan
```

Comment:
do we need to assign different weights for different query types? As that `coprocessor` or `scan` requests can be heavier than point gets.

Reply:
Or we can have a better compare function when ranking topK?

Reply:
Do you have any good suggestions